editor/pot_generator.cpp

```diff
@@ -114,31 +105,43 @@ void POTGenerator::generate_pot(const String &p_file) {
 	_write_to_pot(p_file);
 }
 
-Vector<String> POTGenerator::_parse_scene(const Ref<SceneState> &p_state) {
+void POTGenerator::_parse_scene(const String &p_file_path, Vector<String> *r_extracted_strings) {
```

Comment:
This should be moved to an `EditorTranslationParserPlugin` that handles `PackedScene` resources.